exid-types.json

```
@@ -81,4 +81,28 @@
     "change-controller": "FindAGrave.com",
     "reference": "https://support.findagrave.com/s/article/Cemetery-Search"
   }
+  {
+    "label": "BillionGraves Grave ID",
+    "type": "https://www.billiongraves.com/grave",
```

```suggestion
    "type": "https://www.billiongraves.com/grave/",
```
Addition to make the type + id = URL, as recommended in the spec:

> If the authority maintains stable URLs for each identifier it issues, it is recommended that the [TYPE](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#TYPE) payload be selected such that appending the [EXID](https://gedcom.io/specifications/FamilySearchGEDCOMv7.html#EXID) payload to it yields that URL.

---

In this case the type should be https://www.billiongraves.com/grave/name/ as appending the EXID payload to that gives the resource.